mep/people/management/commands/export_locations.py

```diff
@@ -0,0 +1,119 @@
+"""
+Manage command to export member data for use by others.
+
+Generates a CSV and JSON file including details on every library member in the
+database, with summary details and coordinates for associated addresses.
+
+"""
+
+from collections import OrderedDict
+from django.db.models import Prefetch
+from mep.common.management.export import BaseExport
+from mep.common.templatetags.mep_tags import domain
+from mep.common.utils import absolutize_url
+from mep.people.models import Location, Person
+from mep.accounts.models import Account, Address
+from mep.accounts.partial_date import DatePrecisionField
+
+
+class Command(BaseExport):
+    """Export member data."""
+
+    help = __doc__
+
+    model = Address
+
+    csv_fields = [
+        "id",  # including "id" to store slug for exports,
+        # given not all exported entities have a URI
+        "member_id",  # member slug
+        "member_uri",
+        "care_of_person_id",  # person slug
+        "street_address",
+        "postal_code",
+        "city",
+        "arrrondissement",
+        "country",
+        "start_date",
+        "end_date",
+        "start_date_precision",
+        "end_date_precision",
+        "longitude",
+        "latitude",
+    ]
+
+    def get_queryset(self):
+        """
+        custom filter needed to return person-address combos,
+        so we can pass a one object per row to `get_object_data`
+        """
+        addresses = Address.objects.prefetch_related(
+            Prefetch("account"),
+            Prefetch("person"),
+            Prefetch("location"),
+        )
+        res = []
+        for addr in addresses.all():
+            persons = [addr.person] if addr.person else addr.account.persons.all()
+            for person in persons:
+                res.append((person, addr))
+        return res
```

Comment:
I think there is a lingering possibility in the db structure that we never finished cleaning up which allows an address to be directly associated with a person instead of an account but in practice we should only ever use account.  

Did you find otherwise in the actual data when you were working on the exports? (maybe something changed since the last time I worked on this part)

I remember working on the cleanup but I don't remember where we left it; if need be I can look at the history and figure out more.

Reply:
I found cases where only a person relation existed and instances where only an account relation existed

Reply:
I'm having trouble finding the github issue, but I checked the admin interface and it's not possible to edit or add relationships directly between people and addresses.  I also checked with the python console in production and I am not finding any records with that relationship - can you confirm?  

FWIW, we removed it because we decided it was out of scope for the project, so even if they were there they would arguably be out of scope for this export also.

I can't remember why we removed it from the interface and not from the underlying data structure, maybe that was meant to be a secondary step once the relationships were migrated. I apologize for the confusion in the code, it's certainly not obvious there that you can ignore it.